exporter/collector/googlemanagedprometheus/monitoredresource.go

```diff
@@ -31,23 +31,34 @@ const (
 	namespaceLabel = "namespace"
 )
 
+// promTargetKeys are attribute keys which are used in the prometheus_target monitored resource.
+// It is also used by GMP to exclude these keys from the target_info metric.
+var promTargetKeys = map[string][]string{
+	locationLabel:       {locationLabel, semconv.AttributeCloudAvailabilityZone, semconv.AttributeCloudRegion},
+	clusterLabel:        {clusterLabel, semconv.AttributeK8SClusterName},
+	namespaceLabel:      {namespaceLabel, semconv.AttributeK8SNamespaceName},
+	"job":               {semconv.AttributeServiceName},
```

Comment:
nit: should we make these strings constants above?  Makes typos less likely.

Reply:
Yeah they are used in a couple places. done